templates/blocks/block--bare.html.twig

```diff
@@ -1,17 +1,9 @@
 {#
 /**
  * @file
- * Default theme implementation for a branding block.
+ * Theme override system branding block.
```

Comment:
We could easily get some consistency in the `@file` comment by just replacing the initial part of core's original description, for example:

- `Default theme implementation to display a...` would become `Theme override to display a...`
- `Default theme implementation for a... ` would become `Theme override for a...`

Basically we could just replace `Default theme implementation` with `Theme override`.